strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/config/Maven2LayoutProviderConfig.java

```diff
@@ -12,10 +12,12 @@
 import java.nio.file.FileSystem;
 import java.nio.file.spi.FileSystemProvider;
 
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
 import org.springframework.context.annotation.*;
 import org.springframework.core.env.Environment;
 
 @Configuration
+@ConditionalOnClass(Maven2LayoutProvider.class)
```

Comment:
unnecessary

Reply:
Done. Remove annotations and add last commit PypiLayoutProvider starter